godot-core/src/builtin/vectors/vector_macros.rs

```diff
@@ -575,17 +575,44 @@ macro_rules! impl_float_vector_fns {
                 )
             }
 
-            /// Returns the vector scaled to unit length. Equivalent to `self / self.length()`. See
-            /// also `is_normalized()`.
+            /// Returns the vector scaled to unit length or [`None`], if called on a zero vector.
+            ///
+            /// Equivalent to `self / self.length()`. See also [`normalized()`][Self::normalized] and [`is_normalized()`][Self::is_normalized].
```

Comment:
> Equivalent to `self / self.length()`.

This is not true (for all 4 overloads). I know what you mean here, but calling it equivalent means that the user could replace their code with this, and the result would be the same.

I would mention something like this instead (in all places):

> Computes `self / self.length()`.